src/SUMMARY.md

```diff
@@ -2,7 +2,7 @@
 
 - [Introduction](./introduction.md)
 - [Installation](./installation/index.md)
-  - [AUR](./installation/aur.md)
+  - [Arch](./installation/arch)
```

Comment:
I could also name it `Arch Linux`, let me know and I'll change it and other mentions.

Reply:
Good catch, fixed it